cfg/config.yaml

```diff
@@ -312,4 +314,4 @@ target_mapping:
     - "node"
     - "controlplane"
     - "policies"
-    - "etcd"
+    - "etcd"
```

Comment:
There is a yaml-lint error:
`config.yaml:317:13: [error] no new line character at the end of file (new-line-at-end-of-file)`

https://github.com/aquasecurity/kube-bench/runs/4663081767?check_suite_focus=true

Reply:
@mozillazg 
Hello!
i fixed this issue and pushed the new file
please let me try again)
maybe we could do it before new year like a christmas gift)

Reply:
@mirtov-alexey hi, there are some needless white space:

<img width="417" alt="" src="https://user-images.githubusercontent.com/485054/147729787-96bdec3a-5637-47b1-949d-5de994070f85.png">

https://github.com/aquasecurity/kube-bench/runs/4664605716?check_suite_focus=true

>  Error: onfig.yaml:318:5: [error] no new line character at the end of file (new-line-at-end-of-file)
> Error: onfig.yaml:318:1: [error] trailing spaces (trailing-spaces)
> Error: Process completed with exit code 1.

Reply:
@mozillazg fixed it)sorry its my first pr in public repos